pkg/cloudprovider/provider/azure/provider.go

```diff
@@ -32,7 +32,8 @@ type provider struct {
 	configVarResolver *providerconfig.ConfigVarResolver
 }
 
-type rawConfig struct {
+// RawConfig is a direct representation of an Azure machine object's configuration
+type RawConfig struct {
```

Comment:
I don't see the usage, only exporting ?
Are there any packages that want to use `RawConfig` ?

Reply:
https://github.com/kubermatic/kubermatic/blob/master/api/pkg/resources/machine/machine.go

Reply:
oh, I see.